vault/notebook/five-mindfulness-trainings.md

```diff
@@ -0,0 +1,27 @@
+# Five Mindfulness Trainings
+
+*   _From Thich Nhat Hanh_
```

Comment:
Get this was a copy and paste ... and we probably don't want the bullet point here. These are examples of minor improvements we could make in the transfer over.

Reply:
Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939